scania/corporate-ui-site
# Improvement - add icon modal

**Describe pull-request**  
- Open modal with detail information when clicking on icon.
- Add download button

**Solving issue**  
Part of
Fixes: #92 (second part)

**How to test**  
1. Setup local strapi on this branch: scania/corporate-ui-cms#16
2. Update database
3. Run corporate-ui-site with localhost (fetchData.js is pointing to localhost for now)
4. Check Icon Page following basic layout from Design

**Screenshots**  
_If applicable, add screenshots to help explain_
![image](https://user-images.githubusercontent.com/1199101/94238061-11b73380-ff10-11ea-85a7-c480e8e82dd9.png)
